src/APICalls.js

```diff
@@ -48,5 +49,23 @@ export function findMovie(id) {
       console.log('error Fetching Movie!')
       this.setState({error: 'Your movie has not been found!'})
     })
+}
 
+export function postRating(userId, movieId, userRating) {
+  fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/users/${userId}/ratings`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      movie_id:movieId,
```

Comment:
Oh interesting, noticeably slimmed down from what we had before, did we read the data incorrectly?